luajit21.py

```diff
@@ -323,7 +323,7 @@ def debug_framepc(L, T, fn, pt, nextframe):
     pos = proto_bcpos(pt, ins) - 1
     if pos > pt['sizebc']:
         T = ((ins - 1).cast(typ("char*")) - \
-                typ("GCtrace")['startins'].bitpos / 8).cast(typ("GCtrace*"))
+                int(typ("GCtrace")['startins'].bitpos / 8)).cast(typ("GCtrace*"))
```

Comment:
Look like the left bracket `)` should be after `.bitpos`.

Reply:
Left bracket or ")"? We divide bitpos by 8 and get float number with py3. Likely we want just 8-bit aligned position  (int).

Reply:
@pyhalov
Thanks for your explanation. Now I understand it.